src/modules/announcements/pages/Announcements.jsx

```diff
@@ -1,3 +1,53 @@
+import { useEffect, useState } from "react";
+import { useLocation } from "react-router-dom";
+import { SearchForm } from "../../../components/SearchForm/SearchForm";
+import { ANNOUNCEMENTS_LIST } from "../../../constants/announcements";
+
 export const AnnouncementList = () => {
-  return <div>announcement list</div>;
+  const [filteredAnnouncements, setFilteredAnnouncements] = useState([]);
+  const { search } = useLocation();
+
+  const filterAnnouncements = (data) => {
+    const { Phrase, Category, City, Animals } = data;
+    let announcements = [...ANNOUNCEMENTS_LIST];
```

Comment:
Why destructure `array` into `array`?

Reply:
Isn't it good practice? I operate on new array, not reference

Reply:
Right. I did not pay attention to it.